### PR TITLE
Creation of Footer

### PR DESCRIPTION
**Pull request by ganttArt**
Added footer with website creation attribution and links to social media. Styling for footer in base.css.
Screen Shot 2020-09-28 at 8 45 39 AM

### CODE REVIEW

File: groceries_app/static/css/base.css [hunk 1]
```diff
@@ -1,3 +1,25 @@
-nav{
-    margin: 4px 4px 4px 4px;
+nav {
```
Comment (Azadmel) [inline groceries_app/static/css/base.css:1]: Curious, why is it nav and not `.nav`?

File: groceries_app/static/css/home.css [hunk 2]
```diff
@@ -19,8 +19,6 @@ figcaption {
     /* padding-bottom: 0rem; */
 }
 
-
-
 @media screen and (min-width: 668px) {
```
Comment (Azadmel) [inline groceries_app/static/css/home.css:22]: What does `@media screen and (min-width:668px)` mean?

File: groceries_app/static/images/linkedin.svg [hunk 3]
```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```
Comment (Azadmel) [inline groceries_app/static/images/linkedin.svg:1]: Can you not use a standard image fromat like png/jpeg?
This kinda feels unecessary.

File: templates/base.html [hunk 4]
```diff
@@ -33,9 +33,20 @@
                 </ul>
             </div>
         </nav>
-        {% block content %}
-        {% endblock content %}
-        
+        {% block content %}{% endblock content %}
+        <footer>
+            <div class="card border-primary" id='footer-card'>
```
Comment (Azadmel) [inline templates/base.html:38]: Is `card` a bootstrap class?

**Review by Azadmel — COMMENTED** [review]
Could you add a screenshot to show the visual result of these changes?

File: groceries_app/static/css/base.css (hunk 1 above)
Comment (ganttArt) [reply groceries_app/static/css/base.css:1]: The period is used for classes. Html elements like nav, footer, div, etc. can just be referenced without a period.

File: groceries_app/static/images/linkedin.svg (hunk 3 above)
Comment (ganttArt) [reply groceries_app/static/images/linkedin.svg:1]: Agreed. There can be benefits to using svgs, could be useful for using vector graphics but this is probably not a situation where it matters.

Something I found: "If you know you need vector art, SVG is the choice. Vector art is visually crisp and tends to be a smaller file size than raster graphics like JPG. If you care a lot about the flexibility and responsiveness of the graphic, SVG is the way."

File: groceries_app/static/css/home.css (hunk 2 above)
Comment (ganttArt) [reply groceries_app/static/css/home.css:22]: Everything above a screen width of 668px will have the changes included in this section.

File: groceries_app/static/images/linkedin.svg (hunk 3 above)
Comment (Azadmel) [reply groceries_app/static/images/linkedin.svg:1]: Oh in that case svg does seem to be the better choice.

**Pull request merged**